### ISSUE TITLE
Page feedback form: Allow the form control to be closed so the user can get out of the form field in case they accidentally navigated to. The control just stays open on the screen.

### ISSUE THREAD

**Issue by adam-little** [cagov/design-system#633]
**Context:**

See [Accessibility Review Document](https://docs.google.com/document/d/13FEjWgNYisN_1ahSffZApXYrCw7Kli7Uw1t1qK4YmG8/edit#):
![image](https://user-images.githubusercontent.com/69167298/153071097-9a639d4d-98a0-487c-ae20-5ba7cd1ecf99.png)

**Suggested solution:**

Add "Cancel" button next to "Submit" button. The Cancel button will reset the form.

![image](https://user-images.githubusercontent.com/69167298/153072488-84fb6903-e573-4baa-afbd-bb39cdb0381c.png)

See Figma: https://www.figma.com/file/veQuKQHf8gK5JdFEoMLGRD/UI-Design-kit?node-id=0%3A1

**Comment by adam-little** [comment]
This is ready for development

**Comment by chachasikes** [comment]
I see this. Thanks Adam. Would be good for next sprint. Gonna tag all the forms issues so it's easier to see all the changes. 

@zakiya I'm adding labels on the Forms so we can find these issues.

**Comment by jaredwyee** [comment]
If we update this component also an opportunity to improve how it wraps in a responsive view so that the 'Yes' and 'No' are are a set together vs independently wrapping
![image](https://user-images.githubusercontent.com/85529382/153965205-666feb24-a549-4238-b350-0651f2a001f9.png)

**Comment by adam-little** [comment]
> 

@jaredwyee I explored some approaches here: https://www.figma.com/file/veQuKQHf8gK5JdFEoMLGRD/UI-Design-kit?node-id=89%3A2867

I have sort of lost track of what the actual accessibility issue is. 

It seems like we already meet the criteria of "checked".

FWIW, here are a couple examples of a similar form:
- https://sf.gov/
- https://www.gov.uk/

**Comment by jaredwyee** [comment]
Thanks @adam-little , I think the crux of it was that a user should be able to back out of the text input box flow. 

There's some questions around how and if sites can use the page feedback component so I'm not going to push this forward yet cause I don't want to update the component while other aspects around the data storage / backend is being batted around.

That said, I liked version 3 where there is a visual indication of which selection you made (Yes/No) and a notation that further comment is optional.
